appmap.pth

```diff
@@ -1 +1 @@
-import appmap
\ No newline at end of file
+try: import appmap; except Exception: raise
```

Comment:
How does this change program behavior?  Wouldn't an exception have been raised before if `import appmap` failed?

Reply:
The behavior here is a little strange. Without this change, the python runtime displays a traceback showing where the import failed, but allows the script to continue. This leads to builds passing that should fail, e.g. https://app.travis-ci.com/github/getappmap/appmap-python/builds/256844492#L2780 . In this case, `appmap-agent-init` writes the expected config JSON, even though the import failed.

With the `try ... except: raise` change, `appmap-agent-init` no longer writes the expected JSON, the build fails, and we know we have a problem.

In practice, given that we shouldn't be deploying broken builds, I don't expect the user to ever see the result of this change.

Reply:
I suggest adding a comment there with this explanation.

Reply:
Done.